src/weather.js

```diff
@@ -1,6 +1,6 @@
 require('dotenv').config()
 const fetch = require('node-fetch');
-
+var moment = require('moment');
```

Comment:
I'd use a const for this! https://medium.com/@pandeysoni/when-should-use-const-and-let-instead-of-var-in-javascript-ec2c3d7e5ca6